pages/myContacts/myContacts.js

```diff
@@ -0,0 +1,10 @@
+const myContacts = ({loginName}) => {
+  return (
+    <>
+      {console.log(loginName)}
```

Comment:
No `console.log()` should be included in the PR.

Reply:
Fixed in next pull request.